src/main/java/org/fundacion/jala/converter/core/VideoConverter.java

```diff
@@ -50,27 +50,27 @@ public void convertVideo() throws ConverterException, IOException {
         String ffmpegCommand = startFirstCommand + adaptPath + " ";
         String parameters = changeResolution() + changeFrameRate() + removeAudio();
         String theCommand = ffmpegCommand + parameters + pathOutput + output  + "\" -y";
+        try {
         Process process = Runtime.getRuntime().exec("cmd /c " + theCommand);
         System.out.println(theCommand);
         ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
         errorHandler.start();
         ThreadHandler threadHandler = new ThreadHandler(process.getInputStream(), "Output Stream");
         threadHandler.start();
         LOGGER.info("start");
-        try {
             LOGGER.info("Execute Try");
             process.waitFor();
             LOGGER.info("finish");
```

Comment:
I think these 3 loggers are no longer needed: Start, Execute try and finish

Reply:
You are right. I just fixed it